assets/samples/dynakube/v1beta4/logMonitoring.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: dynatrace.com/v1beta4
```

Comment:
Would it make sense to have a sample that uses this `telemetryService` field?

Reply:
Yes, but not within the scope of this PR. This will be done in scope of the feature development.